### PR TITLE
style: cell island and content grid

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Introduces a new style class to style cells as "Island" and review content grid cell spreading and gap as in new design.

# Note
In `Content` component the option `contrast` is not used anymore in NNS-dapp. However, just in case we would need it again soon, I did not removed it in this PR.

### CODE REVIEW

File: src/lib/components/Island.svelte [hunk 1]
```diff
@@ -20,15 +20,15 @@
...
   .island {
-    background: var(--focus-background);
-    color: var(--focus-background-contrast);
+    @include content.island;
 
     height: 100%;
 
     ::-webkit-scrollbar {
-      background: var(--focus-background);
+      @include content.island;
```
Comment (lmuntaner) [inline src/lib/components/Island.svelte:31]: Why is this needed?

**Review by lmuntaner — APPROVED** [review]
Just a comment. already approving.

File: src/lib/components/Island.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/lib/components/Island.svelte:31]: Because the component and the utility share the same colors. As we don't have component related style in the dark/light/variable scss, I used a mixin to avoid duplication so that if one day we change the island colors, it changes the color in both.

Comment (lmuntaner) [reply src/lib/components/Island.svelte:31]: But why is it also needed inside `::-webkit-scrollbar`? It's already above

Comment (lmuntaner) [reply src/lib/components/Island.svelte:31]: Does it style the scrollbar in the same way of the container and that's needed?

Comment (peterpeterparker) [reply src/lib/components/Island.svelte:31]: It styles the background of the scrollbar. The "color" field of the mixin is not use but including it spare duplicating the background property

Capture d’écran 2022-11-29 à 12 58 20

Comment (lmuntaner) [reply src/lib/components/Island.svelte:31]: ok, thanks for the clarification
